### PR TITLE
put art under testing

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

**Comment by bgruening** [comment]
@erasche any idea what can cause the differences in test data. The dependency should be the same.

**Comment by erasche** [comment]
I was just about to comment on this... I have no idea. Maybe their compilation causes the `--seed` flag to behave different? jeez. Will test locally.

**Comment by bgruening** [comment]
Sorry, you can also leave this to me. Just wanted to know if you have a spontaneous idea.

File: tools/art/test-data/art.454.021.fq [hunk 1]
```diff
@@ -1,1852 +1,1840 @@
 @chr1_1-1
-TCCAGAAATTGTTCCAAGTATCGGCTT
+TCCAGAAATTGTTCCCAAGTATCGGCTTGTTTAC
 +
-:GC?7...IICEECCII6I?BC<<F<@
+FHH:@EEEFF6BBDDD::DI36IGGBCCCFFFCC
```
Comment (bgruening) [inline tools/art/test-data/art.454.021.fq]: can we shorten this?

Comment (erasche) [inline tools/art/test-data/art.454.021.fq]: Absolutely

**Comment by bgruening** [comment]
This is already done!

**Pull request closed without merging**